types/src/guilds/etc.ts

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
`etc.ts`? Maybe could give a better name.

Reply:
Hmm what would be good